src/devices/cpu/es5510/es5510.h

```diff
@@ -12,9 +12,14 @@
 
 #pragma once
 
+// TODO : Not verified
+#define DRAM_SIZE (1<<20)
+#define DRAM_MASK (DRAM_SIZE-1)
```

Comment:
Please don't add constants like this at global scope, and don't use `#define` preprocessor macros.  Make them members of the state class, using either `static constexpr` or `enum`.